newm-server/src/main/kotlin/io/newm/server/features/collaboration/repo/CollaborationRepositoryImpl.kt

```diff
@@ -142,6 +142,18 @@ internal class CollaborationRepositoryImpl : CollaborationRepository {
         }
     }
 
+    override suspend fun invite(songId: UUID) {
+        logger.debug { "invite: songId = $songId" }
+        val emails = mutableListOf<String>()
+        transaction {
+            CollaborationEntity.findBySongId(songId).forEach { collab ->
+                collab.status = CollaborationStatus.Waiting
+                emails += collab.email
+            }
+        }
+        // TODO: Send invitation email to all or only unregistered Collaborators?
```

Comment:
I will do the email part on separate PR after we decide who receives and the email copy itself.  In the meantime, updating to CollaborationStatus.Waiting unblocks the front end testing.